replica/table.cc

```diff
@@ -2582,26 +2582,35 @@ table::make_reader_v2_excluding_sstables(schema_ptr s,
 future<> table::move_sstables_from_staging(std::vector<sstables::shared_sstable> sstables) {
     auto units = co_await get_units(_sstable_deletion_sem, 1);
     sstables::sstable::delayed_commit_changes delay_commit;
+    std::unordered_set<compaction_group*> compaction_groups_to_notify;
     for (auto sst : sstables) {
         try {
             // Off-strategy can happen in parallel to view building, so the SSTable may be deleted already if the former
             // completed first.
             // The _sstable_deletion_sem prevents list update on off-strategy completion and move_sstables_from_staging()
             // from stepping on each other's toe.
             co_await sst->change_state(sstables::normal_dir, &delay_commit);
+            auto& cg = compaction_group_for_sstable(sst);
+            if (sst->requires_cleanup(cg.as_table_state(), sst)) {
```

Comment:
i think you meant to put something like
```c++
sst->requires_cleanup(sst);
```

otherwise we have FTBFS, like:
```
01:00:34  replica/table.cc:2594:22: error: no member named 'requires_cleanup' in 'sstables::sstable'
01:00:34              if (sst->requires_cleanup(cg.as_table_state(), sst)) {
01:00:34                  ~~~~~^
01:00:34  1 error generated.
```

see https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/1112/consoleFull

Reply:
oops, sorry, I had this fixed locally but apparently forgot to amend the commit before pushing :-/